README.md

```diff
@@ -1,4 +1,15 @@
+### SPANISH PLACEMENT EXAM SCORE
+
+The project is about getting the spanish placement exam scores that is taken by students by enrolling to a specific canvas
+course and sending the to Mpathways. This project get grades from canvas using the CanvasAPI and send them to the Mpathway. API directory
+is used in both cases for getting and sending the grades.
+
+### Project Setup
 1. create venv as `python3 -m venv <directory-to-spe-venv>` and activate the venv
```

Comment:
Since you give a detailed explanation of how to create the venv, you might as well include details about how to activate it, too.

Reply:
Added more details